doc/source/ray-core/doc_code/gpus.py

```diff
@@ -86,7 +86,7 @@ def ping(self):
 # __leak_gpus_start__
 # By default, ray will not reuse workers for GPU tasks to prevent
 # GPU resource leakage.
-@ray.remote(num_gpus=1)
+@ray.remote(max_calls=0)
```

Comment:
we still need `num_gpus=1` so that it's a gpu task.

Reply:
Thanks for reminding,  so the correct code is @ray.remote(num_gpus=1, max_calls=0), right?

Reply:
> we still need `num_gpus=1` so that it's a gpu task.

I have modified the corresponding part which could be checked in the  Files changed item , thanks.

Reply:
@jjyao LGTM; you?

Reply:
Yes. @surenyufuz after resolving merge conflicts, we can merge the PR.